main.go

```diff
@@ -32,19 +33,21 @@ type config struct {
 }
 
 const (
-	maxCommitBodyLenght = 72
+	trimEnding = "..."
 )
 
-func printLogAndExportEnv(gitCmd git.Git, format, env string, trim bool) error {
+var maxCommitMessageLength int
```

Comment:
i agree with @BirmacherAkos :
- modify `initCommitMessageLength`: it should just read the envman config and return the maxLenght
- printLogAndExportEnv is only called in the mainE, where you call the current init initCommitMessageLength, so you can easily pass the max message lenght as local variable
- instead of `trimmableVariable bool` input, accept the max env length